events/zurich-9.md

```diff
@@ -1,12 +1,12 @@
 ---
 parent: Events
 title: Machine Translation Meetup
-description: Machine Translation Meetup (MTM) - Zurich 
+description: Machine Translation Meetup (MTM) - Zürich
```

Comment:
IMHO it's just a weird bug in Meetup that they use the ü in English, it's not correct or official (e.g. on the English version of the cities official website).